workers/unity/Assets/Fps/Scripts/Editor/MapBuilderVisualisationWindow.cs

```diff
@@ -60,6 +62,11 @@ public void OnGUI()
                     "Different seeds produce different maps."),
                 seed);
 
+            mapBuilderSettings = (MapBuilderSettings) EditorGUILayout.ObjectField(new GUIContent(
+                    "Map Builder Settings",
```

Comment:
This formatting looks wrong, was this file formatted?

Reply:
yeah I was confused to, I did the code cleanup thing in Rider too